Makd.mak

```diff
@@ -128,17 +128,21 @@ FPM ?= fpm
 # Default dot binary location
 DOT ?= dot
 
+# Default package type
+PKGTYPE ?= deb
+
 # Default flags to pass to graph-deps (see --help for details)
 GRAPH_DEPS_FLAGS ?= -c -C
 
 # Default fpm flags. By default it builds Debian packages from files, a Debian
 # changelog is provided, and a version and iteration (using the Debian version)
 # (defined lazily so we can use target variables)
 ifndef PKG_DEFAULTS
-PKG_DEFAULTS = -D-f -D-sdir -D-tdeb -D--deb-changelog="$O/changelog.Debian" \
+DISTRO_CODENAME="$(shell lsb_release -rs | grep rolling || lsb_release -cs)"
+PKG_DEFAULTS = -D-f -D-sdir -D-t$(PKGTYPE) -D--deb-changelog="$O/changelog.Debian" \
 		-D--version="$(PKGVERSION)" \
-		-D--iteration="$(shell lsb_release -cs)" \
-		-d lsb_release="$(shell lsb_release -cs)"
+		-D--iteration="$(DISTRO_CODENAME)" \
+		-d lsb_release="$(DISTRO_CODENAME)"
```

Comment:
I think current version is actually more reliable. It will work with any distro that reports itself as rolling release in `lsb_release -rs`, not just Arch.